content/en/_index.html

```diff
@@ -204,41 +204,41 @@ <h2 class="section-label">Resources</h2>
-    title="Flux Deep Dive with Stefan Prodan" %}}
-  Stefan walks through what Flux is, who it’s for and its components. Then gives a demo on how to use Flux for delivering infrastructure and applications changes onto Kubernetes.
+    youtube="5u45lXmhgxA"
+    title="Full GitOps Tutorial: Getting started with Flux CD" %}}
+  Anaïs Urlichs walks you through from getting started with Flux to Installing the Starboard Helm Chart through GitOps best practices finally managing our application, setting up notification and monitoring using Flux.
   {{% /blocks/resource %}}
 
   {{% blocks/resource
-    youtube="NLU6b82FhFs"
-    title="From Wild West to Flux Multi-tenancy with Mae Large & Russ Parmer" %}}
-  Mae and Russ share how the State Farm GitOps team helped to simplify application deployment while meeting compliance requirements, enable the admin team to handle scale and embrace the declarative nature of Kubernetes.
+    youtube="PFLimPh5-wo"
+    title="The FASTEST way to deploy apps to Kubernetes" %}}
+  Techno Tim shows how Flux can be leveraged to monitor git repositories, source control, image container repositories, helm repositories, and more.
   {{% /blocks/resource %}}
 
   {{% blocks/resource
-    youtube="gfyo6VqgD_k"
-    title="Flux's Catalog - Keeping Track of Things! with Leigh Capili" %}}
-  Leigh Capili gives us a black belt, under-the-covers look at how Flux's kustomize-controller tracks the child objects for a Kustomization in the Kubernetes API.
+    youtube="cR2eCMbiZg4"
+    title="GitOps with GitHub Actions & Flux with Kingdon Barrett" %}}
+  Kingdon Barret presents a demo of GitHub actions for doing Flux Flux deployment automation using SemVer with GitRepository Source, HelmRelease, and Image Update Automation.
   {{% /blocks/resource %}}
 
   {{% blocks/resource
-    youtube="laMwuG8r7Tw"
-    title="Handling Dependencies with Flux with Jason Morgan" %}}
-  Jason walks through how to use Flux's dependency manager to ensure that your apps get added to your mesh every time.
+    url="https://youtube.com/playlist?list=PLwjBY07V76p53FP85y03psCnb95u095Do"
+    thumbnail="BrYgx4cB7p4"
+    title="Flux at GitOps Con, KubeCon, CloudNativeCon EU 2022" %}}
+  A playlist with a compilation of all talks featuring Flux during GitOpsCon & KubeCon / CloudNativeCon EU 2022 (May 17 – 20).
   {{% /blocks/resource %}}
 
   {{% blocks/resource
-    url="https://www.youtube.com/playlist?list=PLG9qZAczREKmCq6on_LG8D0uiHMx1h3yn"
-    thumbnail="N6UCKF7JD7k"
-    title="Flux Deep Dive series with Geert Baeke" %}}
-  In this 5 video series, Geert Baeke takes a deep dive of Flux and the use of GitOps principles with an Intro to Flux, Intro to Kustomize, Deploying Manifests, Monitoring & Alerting, and Helm Basics.
+    url="https://www.youtube.com/playlist?list=PL9lTuCFNLaD0NVkR17tno4X6BkxsbZZfr"
+    thumbnail="qCDWsIcFU-A"
+    title="GitOps Days 2022" %}}
+  A playlist with a compilation of all talks during GitOps Days 2022 on June 8-9, 2022.
```

Comment:
WDYT about this instead:
Playlist from GitOps Days 2022 on June 8-9, 2022 featuring Flux integrators, end-users/adopters, and maintainers.